src/domain/interfaces/human-description-api.interface.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Expression,
+  HumanReadableFragment,
+  MessageTemplates,
+  ValueType,
+} from '../../datasources/human-description-api/entities/human-description.entity';
+
+export const IHumanDescriptionApi = Symbol('IHumanDescriptionApi');
+
+export interface IHumanDescriptionApi {
```

Comment:
This interface is mostly for the data layer to know which methods it needs to implement. It makes it easier to replace the json implementation that you added with a different one.

I would still create something like a `Repository` which handles business logic regarding the human descriptions – this would be the `domain` class and the one to be used on the `routes` layer (instead of this interface directly).